packages/flutter_localizations/lib/src/l10n/cupertino_tr.arb

```diff
@@ -0,0 +1,22 @@
+{
+  "datePickerHourSemanticsLabelOne": "bir saat",
+  "datePickerHourSemanticsLabelOther": "$hour saat",
+  "datePickerMinuteSemanticsLabelOne": "bir dakika",
+  "datePickerMinuteSemanticsLabelOther": "$minute dakika",
+  "datePickerDateOrder": "dmy",
+  "datePickerDateTimeOrder": "date_time_dayPeriod",
+  "anteMeridiemAbbreviation": "ÖÖ",
+  "postMeridiemAbbreviation": "ÖS",
+  "todayLabel": "Bugün",
+  "alertDialogLabel": "Uyarı",
+  "timerPickerHourLabelOne": "saat",
+  "timerPickerHourLabelOther": "saat",
+  "timerPickerMinuteLabelOne": "dk.",
```

Comment:
Are these how iOS abbreviates them?

Reply:
Yes, I double-checked these using the stock Clock app. 

![IMG_B49FF14C2E9B-1](https://user-images.githubusercontent.com/1281777/57123384-7dc45500-6d79-11e9-82b6-fc00a3c25df7.jpeg)